sklearn/model_selection/_split.py

```diff
@@ -997,6 +997,33 @@ def split(self, X, y=None, groups=None):
             for train_index, test_index in cv.split(X, y, groups):
                 yield train_index, test_index
 
+    def get_n_splits(self, X=None, y=None, groups=None):
+        """Returns the number of splitting iterations in the cross-validator
+
+        Parameters
+        ----------
+        X : object
+            Always ignored, exists for compatibility.
```

Comment:
_RepeatedSplits has the method get_n_splits, which is currently the base class for RepeatedKFold and RepeatedStratifiedKFold only. And those classes pass KFold in the arg cv.
The get_n_splits of KFold has the same documentation.